api/src/pcapi/routes/backoffice_v3/serialization/venues.py

```diff
@@ -21,6 +21,15 @@ class VenueStats(BaseModel):
     total_revenue: float
 
 
+class VenueFinanceData(BaseModel):
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏J'aurais plutôt appelé ça `VenueBankInformation` parce que finance data ça peut aussi bien évoquer les transactions comptables que les informations bancaires.